### ISSUE TITLE
Set Repo Description

### ISSUE THREAD

**Issue by shopify-services** [Shopify/rbi#2]
# Set Repo Description

## TL;DR
Set a description for your service in Slack with spy: `spy github set description :repo *description`

## Why is this being asked?
A repo description makes your service more discoverable in Services DB, especially when it comes to differing internal and external project names.

## What will happen if it doesn't get done within the expected timeframe?
This is mandatory. If this is not done your service will not be easily discoverable in Services DB.

## This doesn't apply to my service. What do I do?
First, leave a comment explaining why it doesn't apply. Then, leave another comment as `/not_applicable`, and close the issue.
If you change your mind, make sure to comment any reason and reopen the issue. Once you've fixed it, services-db will close the issue for you next day.

## I have questions/concerns about this
Please contact the Production Excellence team using Slack at #production-excellence-team.

**Your service**: [rbi/production](https://services.shopify.io/services/rbi/production)
